site/docs/numbers/disconnectNumbers.mdx

```diff
@@ -132,3 +119,6 @@ Content-Type: application/xml; charset=utf-8
 </DisconnectTelephoneNumberOrderResponse>
 
 ```
+
```

Comment:
can you add a section at the bottom and link them to "How to search and order a number"